docs/DeveloperGuide.md

```diff
@@ -3,39 +3,97 @@ layout: page
+**User Target Profile:** Our application is for university students who want to be more financially conscious about their spending habits, enabling them to make wiser economic decisions, providing students on a limited budget with more purchasing power.
 
-**Value Proposition:** It provides university students with an intuitive and frictionless experience to have transparency on their expenditure to help them better understand their spendings so they can develop better financial habits as they transition into adulthood. 
+**Value Proposition:** It provides university students with an intuitive and frictionless experience to have transparency on their expenditure to help them better understand their expenditure so they can develop better financial habits as they transition into adulthood.
+
+{% include toc.html %}
+
+## Terminology
+
+| Term        | Meaning                                                                                           |
+|-------------|---------------------------------------------------------------------------------------------------|
+| Transaction | Represents both an expense or an income. Expenses cause a net loss while incomes cause a net gain |
 
 ## Potential Features:
-- For every user > track individual finances (base)
-- Input spending & set budgets (base)
-- Tag another user and distribute the spendings during outings (extension)
-- Viewing spending habits (extension)
-- Provide price list comparison for all available food outlets near NUS (extension)
-- Allow users to set repeating events (extension)
-- Tagging people for payments gives them a notification (extension)
-
-## Feature List [v1.2]
+
+Potential features are split into two categories, must-haves and good-to-haves. Our focus this project was to flesh out the must-haves first.
+
+### Must-haves
+
+Every user can...
+
+- Track and manage their expenses
+- Manage a budget
+- View their total expenditure by the month/year
+
+### Good-to-haves
+
+Every user can...
+
+- Visualise their spending habits
+- Divide a group spending and tag friends involved in the spending
+- View a price list comparison of all available shops near NUS
+- Set recurring transactions
+
+## Feature Development Tracking
+
+Features were developed over two sprints, each sprint lasting two weeks.
+
+### Milestone v1.2
+
 #### Track Finances
-- Create transactions - Name, date, location, Tag (type/category)
-  - Input either Income or Expense
-- Delete transactions - Delete a single transaction
-  - Remove either an Income or an Expense
-- Clear transactions - Mass delete all transactions
-- Edit transactions - Modify transactions
-- Find transaction - Find a transaction by a given keyword 
-- Tabulate total expenditure  - Sum of all expenses & remaining balance
+
+[//]: # (TODO: Link to each DG/UG entry)
+
+- Add transactions
+- Delete transactions
+- Clear transactions
+- Edit transactions
+- Find transaction
+- Get total expenditure
```

Comment:
Minor grammatical fix

```suggestion
- Add transaction
- Delete transaction
- Clear transactions
- Edit transaction
- Find transactions
- Get total expenditure
```

Reply:
Would be better if _Clear transactions_ was changed to _Clear all transactions_, and _Find transaction_ to _Find transaction(s)_.